lib/Listeners/Notifications/RequestingMember.php

```diff
@@ -74,11 +87,28 @@ public function handle(Event $event): void {
 		}
 
 		$member = $event->getMember();
+		$circle = $event->getCircle();
 
 		if ($event->getType() === CircleGenericEvent::REQUESTED) {
 			$this->notificationService->notificationRequested($member);
 		} else {
 			$this->notificationService->notificationInvited($member);
+
+			if ($member->hasInvitedBy()) {
+				$author = $member->getInvitedBy()->getDisplayName();
+			} else {
+				$author = 'someone';
+			}
```

Comment:
```suggestion
			}
			$author = ucfirst($author);
```

I see in the screen shot that `test` started with lower-case so, we want to start the statement and name in this case with caps.